hrjobcontract/CRM/Hrjobcontract/Import/Parser/Api.php

```diff
@@ -755,4 +750,134 @@ private function validateBenefitsAndDeductions($value_type, $value)  {
     }
     return array('value'=>$outputArray, 'error_message'=>$errorMessage);
   }
+
+  private function setAutoPopulatedFields()  {
```

Comment:
Maybe split these IFs logic into separate private/protected methods which would set $this->_params? The method would be shorter and easier to read.